science/memesuite

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                memesuite
+version             5.1.1
+categories          science biology
+platforms           darwin linux
+license             Noncommercial
+maintainers         uw.edu:cegrant uw.edu:wnoble gmail.com:tlawbailey
+description         Motif analysis toolkit
+long_description    MEME-Suite is a collection of tools for the analysis of \
+                    motifs in DNA and proteins.
+homepage            http://meme-suite.org
+master_sites        http://meme-suite.org/meme-software/macports \
+                    http://alternate.meme-suite.org/meme-software/macports
+distfiles           meme-5.1.1.tar.gz
+
+worksrcdir          meme-${version}
+checksums           rmd160  79512efc3677aead1e76f8e301ca7b7924e9560b \
+                    sha256  2e00c6e38c68bfd90e2402b385d6a1d83e19e6862977fc6e9048e97b27e76daa \
+                    size    46198911
+
+depends_lib         port:expat \
+                    port:ghostscript \
+                    port:perl5 \
+                    port:p5-file-which \
+                    port:p5-html-template \
+                    port:p5-html-tree \
+                    port:p5-json \
+                    port:p5-math-cdf \
+                    port:p5-xml-parser \
+                    port:p5-xml-simple \
+                    port:python27 \
+                    port:py27-gnureadline \
+                    port:zlib
+
+configure.args      --enable-build-libxml2 \
+                    --enable-build-libxslt \
+                    --with-python=${prefix}/bin/python2.7 \
+                    --with-perl=${prefix}/bin/perl5 \
```

Comment:
Since "perl5" is a symlink, you must specify a version of perl directly. Use perl5.28 or perl5.30. You will need change the dependency on "perl5" to also specify the exact version.

Reply:
Done. Thanks.